src/explore-education-statistics-admin/src/pages/release/datablocks/components/DataBlockPageTabs.tsx

```diff
@@ -78,12 +78,8 @@ const DataBlockPageTabs = ({
       releaseId,
     };
 
-    const boundaryLevel = dataBlock.charts[0]?.boundaryLevel ?? undefined;
-    const tableData = await tableBuilderService.getTableData(
-      query,
-      releaseId,
-      boundaryLevel,
-    );
+    const tableData = await tableBuilderService.getTableData(query, releaseId);
+
```

Comment:
Unless I'm missing something I believe that will be picked up in [EES-5402 the admin side of this work](https://dfedigital.atlassian.net.mcas.ms/browse/EES-5402) - would you agree?

Reply:
Not sure as I don't have the full context here. I think you'll want to coordinate with @tomjonesdev.

However, just looking at this change, if you remove the boundary level parameter and don't feed this through to the endpoint, it will stop returning GeoJSON. This would result in all map chart previews being broken in the admin.

I would expect there to be a new endpoint to fetch GeoJSON in a similar way to the how we're doing this in the data blocks. However, if something like this isn't available yet then we need to revert this change and continue to use the current endpoint!

Reply:
Right you are, I'd stepped over into Admin and made changes there along the way which I think need reverting and any further changes will be added in the subsequent ticket as this aims to focus on using the geoJson endpoint on the frontend